src/pages/using-spark.mdx

```diff
@@ -12,7 +12,7 @@ import {
 } from '@sparkdesignsystem/spark-react';
 
 # Using Spark
-<SprkStack itemSpacing="huge">
+<SprkStack itemSpacing="large">
```

Comment:
Not my whitespace!

Reply:
There's not a big difference between "large" and "huge" white space - first image is the "large" and second image is the "huge". Choosing the large helps the content flow better :)

<img width="991" alt="Screen Shot 2020-10-12 at 4 06 05 PM" src="https://user-images.githubusercontent.com/26862826/95786348-56c7bd80-0ca5-11eb-8c72-8202bf93c4f6.png">
<img width="981" alt="Screen Shot 2020-10-12 at 4 05 57 PM" src="https://user-images.githubusercontent.com/26862826/95786352-57f8ea80-0ca5-11eb-8201-02a4a786ab71.png">